<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPage</class>
 <widget class="QDialog" name="NewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>750</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>1400</x>
       <y>0</y>
       <width>111</width>
       <height>721</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="Add_tab">
        <property name="text">
         <string>Add Tab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Delet_tab">
        <property name="text">
         <string>Delete tab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Clear_tab">
        <property name="text">
         <string>Clear Tab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Save">
        <property name="text">
         <string>Save </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1400</width>
       <height>720</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
